Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -108,7 +110,22 @@ static Result<NonnullRefPtr<DynamicLoader>, DlErrorMessage> map_library(const St
         return map_library(name, fd);
     }
 
-    // TODO: Do we want to also look for libs in other paths too?
+    // Scan the LD_LIBRARY_PATH envvar if it exists or we're not a SUID program
```

Comment:
The logic below is not actually concerned with whether we are SUID/`AT_SECURE` or not (as that is handled by other parts of the code), so the comment likely shouldn't be concerned with that either.


```suggestion
    // Scan the LD_LIBRARY_PATH environment variable if applicable
```